doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -165,13 +165,13 @@ and ``coqtop``, unless stated otherwise:
 :-load-vernac-object *qualid*: Load |Coq| compiled library :n:`@qualid`. This
   is equivalent to running :cmd:`Require` :n:`qualid`.
 :-ri *qualid*, -require-import *qualid*: Load |Coq| compiled library :n:`@qualid` and import it.
-  This is equivalent to running :cmd:`Require Import` :n:`@qualid`.
+  This is equivalent to running :cmd:`Require` :n:`Import @qualid`.
 :-re *qualid*, -require-export *qualid*: Load |Coq| compiled library :n:`@qualid` and transitively import it.
-  This is equivalent to running :cmd:`Require Export` :n:`@qualid`.
+  This is equivalent to running :cmd:`Require` :n:`Export :n:`@qualid`.
```

Comment:
```suggestion
  This is equivalent to running :cmd:`Require` :n:`Export @qualid`.
```
That being said, would it be possible to preserve the index entries `Require Import` and `Require Export`?